docs/source/guide/quickstart.rst

```diff
@@ -31,7 +31,22 @@ Python. For details, including the deprecation schedule and how to update your
 project to use Python 3.8, see :ref:`guide_migration_py3`.
 
 For information about how to get the latest version of Python, see the official `Python
-documentation <https://www.python.org/downloads/>`_. 
+documentation <https://www.python.org/downloads/>`_.
+
+Setup a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have a supported version of Python installed, you should set up
+your workspace by creating a virtual environment and activate it::
+
+    $ python -m venv .venv
```

Comment:
`venv` currently exists in our .gitignore file, I think we should add `.venv` in there too.

Reply:
Addressed in https://github.com/boto/boto3/pull/4440.